doc/source/installation.rst

```diff
@@ -2,11 +2,222 @@
 Installation
 ============
 
-At the command line::
+Using pip
+---------
+Use::
 
-    $ pip install python-redfish
+    sudo pip install python-redfish
 
-Or, if you have virtualenvwrapper installed::
+Pip will install :
+
+1. The library and all dependencies into your site-packages directory
+2. Redfish client master conf file into /etc/redfish_client.conf
+3. Data file (templates) into /usr/share/redfish-client/templates
+
```

Comment:
I don't know if this is the way pip works normally, but if not, I'd recommend that we install by default under /usr/local, not in the std / FS ! This should be left to packages, not file based installs.